examples/BUILD

```diff
@@ -25,3 +21,21 @@ filegroup(
         ],
     ),
 )
+
+filegroup(
+    name = "certs",
+    srcs = glob(
+        [
+            "_extra_certs/*.pem",
```

Comment:
Tiny nit: I really prefer compact syntax for stuff like this, but it is a bit inconsistent around Envoy. E.g.
```
  srcs = glob(["_extra_certs/*.pem"]),
```
I think it makes things a bit more readable and aligns with a preference for vertical readability in our C++ guide for implementation code.

Reply:
> I really prefer compact syntax

in general me too - ill update...